### ISSUE TITLE
Binary files in repository

### ISSUE THREAD

**Issue by janisozaur** [discordapp/discord-rpc#48]
This repository seems to contain multiple binaries. Such files are not meant to be stored in a git repository, they don't compress very well and degrade git performance.

I would urge to stop storing them here and rather rely on fetching (manually or automatically) released versions from https://github.com/discordapp/discord-rpc/releases or using Git-LFS fallback (but that comes with some limitations on GitHub when used with free tier).

**Comment by crmarsh** [comment]
Yeah, it's problematic. I was trying to keep the unity example simple (checkout the repo and it just works).
